broker/src/main/java/org/apache/rocketmq/broker/BrokerStartup.java

```diff
@@ -64,7 +64,7 @@ public static BrokerController start(BrokerController controller) {
             controller.start();
 
             String tip = "The broker[" + controller.getBrokerConfig().getBrokerName() + ", "
-                + controller.getBrokerAddr() + "] boot success. serializeType=" + RemotingCommand.getSerializeTypeConfigInThisServer();
+                    + controller.getBrokerAddr() + "] boot success. serializeType=" + RemotingCommand.getSerializeTypeConfigInThisServer();
```

Comment:
I think it's better not changing the format of old code without logical modification.